src/storage/hummock_sdk/src/key.rs

```diff
@@ -612,7 +612,7 @@ impl<T: AsRef<[u8]>> FullKey<T> {
     pub fn from_user_key(user_key: UserKey<T>, epoch: HummockEpoch) -> Self {
         Self {
             user_key,
-            epoch_with_gap: EpochWithGap::new(epoch, 0),
+            epoch_with_gap: EpochWithGap::from_u64(epoch),
```

Comment:
Why change to `from_u64`? Looks like the `new(epoch, 0)` is better.

Reply:
https://github.com/risingwavelabs/risingwave/blob/e3c8649ec4b3c5adb3e37d1dd82e8e421be1f8ee/src/storage/src/hummock/sstable/builder.rs#L441 may pass HummockEpoch::MAX here, but I think it can be replaced by `MAX_EPOCH`, whose last 16 bit is 0.

Reply:
Replace all usage of `HummockEpoch::MAX` by `MAX_EPOCH`.